### PR TITLE
Add contacts to groups schema

### PR DESCRIPTION
**Pull request by cmungall**

### CODE REVIEW

File: metadata/groups.schema.yaml [hunk 1]
```diff
@@ -19,3 +19,26 @@ sequence:
         type: str
         required: true
         unique: true
+      "active_from":
+        type: str
+        required: false
+        pattern: /^\d\d\-\d\d\-\d\d\d\d$/
```
Comment (kltm) [inline metadata/groups.schema.yaml]: Would it be possible to get that reversed for iso 8601?

**Comment by kltm** [comment]
It might be good to talk about changes like this.
For the schema, what is an `emaul`? Also:
https://github.com/geneontology/go-site/pull/412/files
Would it be possible to get the date reversed for iso 8601?
Also, for some types of sensitive information, we have go-internal; historically we've had email address count as sensitive.

**Comment by kltm** [comment]
Also, is the intention to have this folded in as well?
https://github.com/geneontology/go-site/issues/231

File: metadata/groups.schema.yaml (hunk 1 above)
Comment (cmungall) [inline metadata/groups.schema.yaml]: +1

**Comment by cmungall** [comment]
yep, can go ahead and remove email (we only have gh in tsv so far) and fix dates iso was intent

**Pull request merged**